lib/matplotlib/figure.py

```diff
@@ -913,6 +913,42 @@ def _break_share_link(ax, grouper):
         # Break link between any twinned axes
         _break_share_link(ax, ax._twinned_axes)
 
+    def clf(self, keep_observers=False):
+        """
+        Clear the figure.
+        Set *keep_observers* to True if, for example,
+        a gui widget is tracking the Axes in the figure.
+        """
```

Comment:
Please use [numpydoc](https://matplotlib.org/devdocs/devel/documenting_mpl.html#example-docstring) for docstrings.